csrc/device_lower/pass/index.h

```diff
@@ -120,7 +120,7 @@ class IndexLowering : private OptOutConstDispatch {
 
   Val* lowerDstIndex(
       Val* dst,
-      const std::unordered_map<int, Val*>& override_index = {},
+      const std::unordered_map<IterDomain*, Val*>& override_index = {},
```

Comment:
This is just an interface change. `lowerSrcIndex` already uses this interface. Previously, overriding wasn't implemented for the consumer indexing case.